README.md

```diff
@@ -111,7 +116,25 @@ Write a conditional statement that...
 ### Your solution here:
 8.  Write your javascript solution below
 ```js
-// your answer here
+let total;
+let num;
+
+
+total = "";
+num = 20;
+
+
+if (num  % 3 === 0) {
+  total = total + "Fizz";
+}
+if (num % 5 === 0) {
+  total += "Buzz";
+}
+if (num % 3 && 5 === 0) {
```

Comment:
this line "if (num % 3 && 5 === 0)" causes a logical error. Also it's not needed. you can delete it